app/assets/javascripts/discourse/components/desktop-notification-config.js.es6

```diff
@@ -0,0 +1,61 @@
+export default Ember.Component.extend({
+  classNames: ['controls'],
+
+  _version: 0,
+
+  _init: function() {
+    this.invalidate();
+  }.on('init'),
+
+  invalidate() {
+    this.set('_version', this.get('_version') + 1);
```

Comment:
Oh, `propertyDidChange` is the thing I was missing. OK! Will get that fixed when I get home